src/Collections/Artemis.Plugins.Input/DataModelExpansion/DataModels/InputDataModel.cs

```diff
@@ -93,4 +116,18 @@ public MouseEventArgs(MouseButton button, string device)
         [DataModelProperty(Description = "The name of the device on which the key was pressed")]
         public string Device { get; set; }
     }
+
+    public class MouseScrollEventArgs : DataModelEventArgs
+    {
+        public MouseScrollEventArgs(int delta, string device)
+        {
+            Delta = delta;
+            Device = device;
+        }
+
+        [DataModelProperty(Description = "Amount of scroll steps")]
+        public int Delta { get; set; }
+        [DataModelProperty(Description = "The name of the device on which the key was pressed")]
```

Comment:
Small mistake here in the description, should be something like "The name of the device which was used to scroll"